openid-4-verifiable-presentations-1_0.md

```diff
@@ -1549,6 +1549,96 @@ Note: The reason hashes of the user claims are included in the `issuerAuth` item
 
 The example in this section is also applicable to the electronic identification Verifiable Credentials expressed using data models defined in ISO/IEC TR 23220-2.
 
+## IETF SD-JWT VC
+
+This section defines how Credentials complying with [@!I-D.ietf-oauth-sd-jwt-vc] can be presented to the Verifier using this specification.
+
+### Format Identifier
+
+The Credential format identifier is `vc+sd-jwt`.
+
+#### Example Credential
+
+The following is a non-normative example of the unsecured payload of an IETF SD-JWT VC that will be used throughout this section:
+
+<{{examples/credentials/sd_jwt_vc_unsecured.json}}
+
+The following is a non-normative example of an IETF SD-JWT VC using the unsecured payload above, containing claims that are selectively disclosable.
```

Comment:
```suggestion
The following is a non-normative example of an Issuer-Signed JWT of an IETF SD-JWT VC using the unsecured payload above, containing claims that are selectively disclosable.
```

Reply:
This suggestion introduces a tautology imho as a VC (Verifiable Credential) is already defined as

> An Issuer-signed Credential whose authenticity can be cryptographically verified.

Reply:
I was trying to differentiate between SD-jwt, disclosures and KB-jwt. The terminology is hard here

Reply:
agree the terminology is hard but think `IETF SD-JWT VC` works here